decidim-core/config/locales/ca.yml

```diff
@@ -48,6 +48,7 @@ ca:
       image_too_big: La imatge és massa gran
   date:
     formats:
+      datepicker: "%d/%m/%Y"
```

Comment:
Again, you should only modify `en.yml` and Crowdin will take care of the rest. This way, every locale gets the chance to potentially set a new format.

Reply:
Oh, I thought that translations were manual and was trying to avoid formatting failures from other languages than the "officially supported". I'll rollback then the moving of date format from admin module to core for other languages than English. Correct?